.github/workflows/preview-fern-docs.yml

```diff
@@ -0,0 +1,61 @@
+name: Generate Fern Docs Preview
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+
+    steps:
+      - name: Checkout aa-sdk
+        uses: actions/checkout@v4
+
+      - name: Download Alchemy Docs Repo
```

Comment:
can this not be done with the github checkout action?

Reply:
TIL you can checkout a repo into a subdirectory. That's way faster